orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -206,6 +209,7 @@ def __init__(self, model, model_desc, optim_config, loss_fn=None, options=None):
 
         self._train_step_info = TrainStepInfo(self.optim_config)
         self._training_session = None
+        self._load_state_dict = None
```

Comment:
I think this is just github hiding the real class this line belongs to. It is written in the ```__init__``` method of ```ORTTrainer``` class. But it appears as though it was written for ```TrainStepInfo``` because of the hidden lines.